### PR TITLE
Rsvp styling updates

### PR DESCRIPTION
**Pull request by shreya317**

### CODE REVIEW

File: app/assets/javascripts/components/rsvp_show.es6 [hunk 1]
```diff
@@ -2,7 +2,17 @@ const RsvpShow = React.createClass({
 
   renderEventName(eventId) {
     let events = this.props.events
-    return events[eventId]
+    return events[eventId].event_name
+  },
+
+  renderEventInfo(eventId) {
+    let events = this.props.events
+    var e = events[eventId]
```
Comment (pdadhaniya) [inline app/assets/javascripts/components/rsvp_show.es6]: all the `var`'s should be `let`

File: app/assets/javascripts/components/rsvp_show.es6 [hunk 2]
```diff
@@ -62,6 +72,7 @@ const RsvpShow = React.createClass({
   renderFamilyMessage() {
     return (
       <div className="row">
+        <p><i className="fi-info"></i>  Hover over event name for details.</p>
```
Comment (pdadhaniya) [inline app/assets/javascripts/components/rsvp_show.es6:71]: space between `</i>` and `Hover`

File: app/controllers/rsvp_controller.rb [hunk 3]
```diff
@@ -10,7 +10,7 @@ def show
     guest = Guest.where(full_name: name, zip: zip)
 
     if guest.empty?
-      flash.now[:guest_not_found] = "Name or ZIP Code does not match, please try again."
+      flash.now[:guest_not_found] = "Name or ZIP Code does not match, please try again or contact the host."
```
Comment (pdadhaniya) [inline app/controllers/rsvp_controller.rb]: `contact Parag or Shreya`?

File: app/views/rsvp/index.html.erb [hunk 4]
```diff
@@ -1,6 +1,8 @@
 <% if flash[:guest_not_found] %>
-  <div class="row">
-    <h3 class="not_found"><%=flash[:guest_not_found]%></h3>
-  </div>
+  <div class="not_found">
```
Comment (pdadhaniya) [inline app/views/rsvp/index.html.erb:2]: The tabbing is 4 spaced here

File: app/assets/javascripts/components/rsvp_show.es6 (hunk 2 above)
Comment (shreya317) [inline app/assets/javascripts/components/rsvp_show.es6:71]: @pdadhaniya space is intentional, otherwise icon touches the text.

**Comment by pdadhaniya** [comment]
:+1:

**Pull request merged**